main.go

```diff
@@ -58,3 +59,18 @@ func postTasks(c *gin.Context) {
 	tasks = append(tasks, newTask)
 	c.IndentedJSON(http.StatusCreated, newTask)
 }
+
+func deleteTask(c *gin.Context) {
+	id := c.Param("id")
```

Comment:
what happens if the id passed in is not in the right 'type' or just does not exist?